src/components/visualizations/chart/test/loadEvent.spec.ts

```diff
@@ -0,0 +1,84 @@
+// (C) 2007-2018 GoodData Corporation
+import handleChartLoad from '../events/load';
+
+const buildChart = (
+    axisGroupAddClass: () => {},
+    labelGroupAddClass: () => {},
+    opposite: boolean
+) => ({
+    yAxis: [
+        {
+            axisGroup: {
+                addClass: axisGroupAddClass
+            },
+            labelGroup: {
+                addClass: labelGroupAddClass
+            },
+            opposite
+        }
+    ]
+});
+
+const addSecondaryAxisToChart = (
+    chart: any
+) => {
+    const { yAxis: [ primaryAxis ] } = chart;
+    return {
+        yAxis: [
+            primaryAxis,
+            {
+                ...primaryAxis,
+                opposite: !primaryAxis.opposite
+            }
+        ]
+    };
+};
+
+const executeTest = (opposite: boolean, isDualAxesChart: boolean) => {
+    const axisGroupAddClass = jest.fn();
+    const labelGroupAddClass = jest.fn();
+
+    let chart = buildChart(axisGroupAddClass, labelGroupAddClass, opposite);
+    if (isDualAxesChart) {
+        chart = addSecondaryAxisToChart(chart);
+    }
+
+    const boundHandleChartLoad = handleChartLoad.bind(chart);
+    boundHandleChartLoad();
+
+    return {
+        axisGroupAddClass,
+        labelGroupAddClass
+    };
+};
+
+describe('handleChartLoad()', () => {
+    // tslint:disable-next-line:max-line-length
+    it('should call addClass() axis methods with classname saying it\'s primary axis if \'opposite\' property is falsy', () => {
+        const { axisGroupAddClass, labelGroupAddClass } = executeTest(false, false);
+
+        expect(axisGroupAddClass).toBeCalledWith('s-highcharts-primary-yaxis');
+        expect(labelGroupAddClass).toBeCalledWith('s-highcharts-primary-yaxis-labels');
+    });
+
+    // tslint:disable-next-line:max-line-length
+    it('should call addClass() axis methods with classname saying it\'s secondary axis if \'opposite\' property is truthy', () => {
+        const { axisGroupAddClass, labelGroupAddClass } = executeTest(true, false);
+
+        expect(axisGroupAddClass).toBeCalledWith('s-highcharts-secondary-yaxis');
+        expect(labelGroupAddClass).toBeCalledWith('s-highcharts-secondary-yaxis-labels');
+    });
+
+    // tslint:disable-next-line:max-line-length
+    it('should call addClass() axis methods with classname on dual axes chart', () => {
+        const { axisGroupAddClass, labelGroupAddClass } = executeTest(false, true);
+
+        expect(axisGroupAddClass).toHaveBeenCalledTimes(2);
+        expect(axisGroupAddClass).toHaveBeenCalledWith('s-highcharts-primary-yaxis');
```

Comment:
Please use `toHaveBeenNthCalledWith` Jest function (https://jestjs.io/docs/en/expect#tohavebeennthcalledwithnthcall-arg1-arg2-).

Reply:
we are using Jest v20.x, so it does not support `toHaveBeenNthCalledWith` yet

Reply:
Allright, sorry.